mirrorbutton.js

```diff
@@ -1,18 +1,36 @@
 // Global
+const TKNMRB_MODULE_ID = "token-mirror-button";
+const TKNMRB_MODULE_NAME = "Token Mirror Button";
```

Comment:
Nitpick: Revert `TKNMRB_MODULE_ID` and `TKNMRB_MODULE_NAME` back to `MODULE_NAME` and `MODULE_ID`. Just my preferred naming lol

Reply:
I have done as requested, but I'm not entirely sure where that's the correct thing to do. Granted I'm not very well-versed with how FoundryVTT actually imports the different module codebase, but is it not all imported into the same global scope, so global variables and constants should have names that uniquely identify them to modules, as not to overwrite similar expressions from other packages?

Reply:
No problem! Foundry uses modules so every .js file has a separate scope unless you explicitly import things